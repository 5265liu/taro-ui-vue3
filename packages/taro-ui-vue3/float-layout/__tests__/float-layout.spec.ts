import { mount } from '@vue/test-utils'
import AtFloatLayout from '../src/index.vue'

const AXIOM = 'Rem is the best girl'

describe('AtFloatLayout.vue', () => {
  test('render test', () => {
    const wrapper = mount(AtFloatLayout, {
      slots: {
        default: AXIOM,
      },
    })
    expect(wrapper.text()).toEqual(AXIOM)
  })
})
