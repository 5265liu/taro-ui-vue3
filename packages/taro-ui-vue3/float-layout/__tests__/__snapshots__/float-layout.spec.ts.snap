// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FloatLayout Snap render FloatLayout -- props isOpened 1`] = `
<view
  class="at-float-layout--active at-float-layout"
>
  <!-- overlay -->
  <view
    class="at-float-layout__overlay"
  />
  <!-- container layout -->
  <view
    class="at-float-layout__container layout"
  >
    <!-- header -->
    <!--v-if-->
    <!-- body -->
    <view
      class="layout-body"
    >
      <scroll-view
        class="layout-body__content"
        scroll-with-animation="false"
        scroll-x="false"
        scroll-y="true"
      >
        
        这是内容区 随你怎么写这是内容区 随你怎么写这是内容区
随你怎么写这是内容区 随你怎么写这是内容区 随你怎么写这是内容区
随你怎么写
        
      </scroll-view>
    </view>
  </view>
</view>
`;

exports[`FloatLayout Snap render FloatLayout -- props title 1`] = `
<view
  class="at-float-layout--active at-float-layout"
>
  <!-- overlay -->
  <view
    class="at-float-layout__overlay"
  />
  <!-- container layout -->
  <view
    class="at-float-layout__container layout"
  >
    <!-- header -->
    <view
      class="layout-header"
    >
      <text
        class="layout-header__title"
      >
        这是个标题
      </text>
      <view
        class="layout-header__btn-close"
      />
    </view>
    <!-- body -->
    <view
      class="layout-body"
    >
      <scroll-view
        class="layout-body__content"
        scroll-with-animation="false"
        scroll-x="false"
        scroll-y="true"
      >
        
        这是内容区 随你怎么写这是内容区 随你怎么写这是内容区
随你怎么写这是内容区 随你怎么写这是内容区 随你怎么写这是内容区
随你怎么写
        
      </scroll-view>
    </view>
  </view>
</view>
`;

exports[`FloatLayout Snap render initial FloatLayout 1`] = `
<view
  class="at-float-layout"
>
  <!-- overlay -->
  <view
    class="at-float-layout__overlay"
  />
  <!-- container layout -->
  <view
    class="at-float-layout__container layout"
  >
    <!-- header -->
    <!--v-if-->
    <!-- body -->
    <view
      class="layout-body"
    >
      <scroll-view
        class="layout-body__content"
        scroll-with-animation="false"
        scroll-x="false"
        scroll-y="true"
      >
        
        这是内容区 随你怎么写这是内容区 随你怎么写这是内容区
随你怎么写这是内容区 随你怎么写这是内容区 随你怎么写这是内容区
随你怎么写
        
      </scroll-view>
    </view>
  </view>
</view>
`;
